extensions/arc/package.json

```diff
@@ -601,37 +609,69 @@
                           "type": "kube_storage_class",
                           "required": true
                         },
-                        {
-                          "label": "%arc.cores-request.label%",
-                          "description": "%arc.sql.cores-request.description%",
-                          "variableName": "AZDATA_NB_VAR_SQL_CORES_REQUEST",
-                          "type": "number",
-                          "min": 1,
-                          "required": false
-                        },
                         {
                           "label": "%arc.cores-limit.label%",
                           "description": "%arc.sql.cores-limit.description%",
                           "variableName": "AZDATA_NB_VAR_SQL_CORES_LIMIT",
                           "type": "number",
                           "min": 1,
-                          "required": false
+                          "required": false,
+                          "validations": [
+                            {
+                              "type": "is_integer",
```

Comment:
Ensure that field isInteger

Reply:
azdata allows fractional values so I am happy to revert and I can bring this back once azdata starts enforcing this as it is the right thing to do. In a chat with PMs in Travis had mentioned that we should allow only integral values for MIAA as they are needed for billing purposes. Adding Travis here. Happy to revert for now and we can bring it back when azdata starts validating.

Reply:
Removed the is_integer checks.

Reply:
PG currently supports fractional cores, though it could change.  We do want integer validation on AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_WORKERS though.

Reply:
Sure. Should I include that check as part of this PR?

Reply:
How about AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_PORT?

Reply:
Added is_integer checks for AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_WORKERS and AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_PORT